Assets/Scripts/Enemy.cs

```diff
@@ -44,12 +53,24 @@ private void Awake()
         rigid = GetComponent<Rigidbody2D>();
         spriteRenderer = GetComponent<SpriteRenderer>();
         anima = GetComponent<Animator>();
-        StartCoroutine("ChangeMovement");
+        if (boss)

```

Comment:
"boss" might be to vague of a variable name. If boss what? If boss is present/enabled? If this object is a boss? Renaming could make this more clear